sim/core/stats/stats.go

```diff
@@ -372,6 +372,7 @@ type PseudoStats struct {
 	CastSpeedMultiplier   float64
 	MeleeSpeedMultiplier  float64
 	RangedSpeedMultiplier float64
+	AttackSpeedMultiplier float64
```

Comment:
I think `HasteMultiplier` would be clearer than `AttackSpeedMultiplier`. Also add a comment here clarifying what this means.